### PR TITLE
Multiple myeloma cause model updates

### PR DESCRIPTION
**Pull request by zmbc**
This is a superset of #899, so the diff won't look good until that is merged.

I do not yet have any data (even placeholder data) generated for these changes. Can we get away with using phase 1 data, interpreting hazard of PFS as hazard of relapse (not subtracting anymore)?

### CODE REVIEW

**Review by aflaxman — APPROVED** [review]
Did you already do a pull request like this? It seems really familiar!

File: docs/source/gbd2019_models/causes/neoplasms/multiple_myeloma_2/cancer_model.rst [hunk 1]
```diff
@@ -450,117 +482,30 @@ baseline hazard rates at the individual level according to individual simulant
...
+   model because we are not interested in the screening and early managment for
+   MM. As a result, the simulation will not track/model simulants with asymptomatic
+   condition.
+#. YLLs are substantially larger than YLDs for this cause. For now, we will not
+   build a disability component to capture those secondary outcomes.
+#. Based on available data, the most advanced disease state in cause model is
+   fourth or higher relapse of RRMM. We intend to track deaths from simulants
+   who have developed fourth relapse and received fifth-line of treatment but ignore
+   the incident cases from fourth relapse to higher relapse of RRMM. As a result,
+   we will not calculate progress-free survival among simulants with fourth or
```
Comment (plinds) [inline docs/source/gbd2019_models/causes/neoplasms/multiple_myeloma_2/cancer_model.rst:507]: I'm assuming progress-free here = progression-free?

File: docs/source/gbd2019_models/causes/neoplasms/multiple_myeloma_2/cancer_model.rst [hunk 2]
```diff
@@ -450,117 +482,30 @@ baseline hazard rates at the individual level according to individual simulant
...
+   of symptomatic MM. That said, we are comfortable using GBD incidence of MM
+   as the detection rate of symptomatic MM cases. The incidence of RRMM will be
+   calculated from survival regression analysis based on Flatiron data.
+#. The asymptomatic/idolent state (smoldering MM) is excluded from this cause
+   model because we are not interested in the screening and early managment for
+   MM. As a result, the simulation will not track/model simulants with asymptomatic
+   condition.
+#. YLLs are substantially larger than YLDs for this cause. For now, we will not
+   build a disability component to capture those secondary outcomes.
+#. Based on available data, the most advanced disease state in cause model is
```
Comment (plinds) [inline docs/source/gbd2019_models/causes/neoplasms/multiple_myeloma_2/cancer_model.rst:503]: in *this* cause model

**Review by plinds — APPROVED** [review]

File: docs/source/gbd2019_models/causes/neoplasms/multiple_myeloma_2/cancer_model.rst (hunk 1 above)
Comment (zmbc) [reply docs/source/gbd2019_models/causes/neoplasms/multiple_myeloma_2/cancer_model.rst:507]: Yes, although it should really be "relapse-free" -- I think I missed a few things in this limitations section that should have been updated from Phase 1. Thanks for pointing this out, I will fix 👍

**Pull request merged**